tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -865,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
-      <div>
-        {this.state.bar.qux}
-      </div>
-    );
+    return <div>{this.state.bar.qux}</div>;
```

Comment:
I ran both options (break if an element contains an expression vs. break if an element contains more than one expression) over the Geckoboard codebase to see how they compared.

I noticed that always breaking on expressions caused some less than ideal layout in a few cases:

```
You currently have <strong>{dashboardStr}</strong> and<strong>{userStr}</strong>
```

becomes:

```
You currently have{' '}
<strong>
  {dashboardStr}
</strong>{' '}
and{' '}
<strong>
  {userStr}
</strong>
```

I'm not sure yet how we could avoid the above output will still aiming to break elements that contain a single expression. Any ideas greatly appreciated!